src/DSP.jl

```diff
@@ -2,14 +2,37 @@ __precompile__()
 
 module DSP
 
+# We want to be very sure we don't pull in Base names unless we're very sure we want them
+# This macro will be called in each submodule herein to do the appropriate imports
+macro importffts()
+    quote
+        using AbstractFFTs, FFTW
+        importall AbstractFFTs, FFTW
+        if VERSION >= v"0.7.0-DEV.585"
```

Comment:
I know, it was a guess. I'm going to go back and change the explicit version checks to be exact immediately once the Base PR merges. The problem with `isdefined` is that I was pretty thorough in the deprecation to ensure that everything was still defined.